src/Avalonia.Base/Media/Color.cs

```diff
@@ -459,11 +459,18 @@ public override string ToString()
         /// <returns>
         /// The integer representation of the color.
         /// </returns>
-        public uint ToUint32()
+        public uint ToUInt32()
         {
             return ((uint)A << 24) | ((uint)R << 16) | ((uint)G << 8) | (uint)B;
         }
 
+        /// <inheritdoc cref="Color.ToUInt32"/>
+        [Obsolete("Use Color.ToUInt32() instead.")]
+        public uint ToUint32()
```

Comment:
I would recommend adding `EditorBrowsable(EditorBrowsableState.Never)` as well to unlist this API from IDE auto-completion list.

Reply:
Not a bad idea but devs should generally watch the build output amd warnings already to catch stuff like obsolete usage. The PR is already merged, go ahead and open a new one if you think it's important enough. We don't usually do this for other Obsolete members from what I recall though.